msteams-platform/sbs-create-dashboard-widget-graph-api-call-in-Teams-toolkit.yml

```diff
@@ -0,0 +1,280 @@
+### YamlMime:Tutorial
+title: Build a dashboard in Teams
+metadata:
+  title: Build a dashboard in Teams
+  description: Learn how to create a dashboard tab, add a new widget, and implement Graph API.
+  audience: Developer
+  level: Beginner
+  ms.date: 01/06/2023
+  ms.topic: interactive-tutorial
+  nextTutorialHref: tabs/how-to/dashboard-teams-toolkit.md
+  nextTutorialTitle: Dashboard, widget, and Graph API call in Teams toolkit
+  ms.custom: mvc
+  ms.localizationpriority: high
+items:
+- durationInMinutes: 1
+  content: |
+    This step-by-step guide helps you to create a dashboard tab, add a new widget to the dashboard, and implement Graph API. You'll see the following output:
+
+    :::image type="content" source="assets/images/sbs-create-a-new-dashboard/dashboard1.png" alt-text="Screenshot shows the final output of dashboard.":::
```

Comment:
The `dashboard1.png` image seems not very clear, I have a clearer picture, please feel free to replace it.

![image](https://user-images.githubusercontent.com/107838226/215257271-dec10690-25c2-4924-96fd-c53fdf600723.png)

Reply:
Updated the image.